pkgs/by-name/sa/satysfi/package.nix

```
@@ -30,14 +30,18 @@ let
     inherit (ocamlPackages.yojson) meta;
   };
 in
-ocamlPackages.buildDunePackage rec {
+let
+  version = "0.0.11";
```

```suggestion
  ocamlPackages = ocaml-ng.ocamlPackages_4_14;

  version = "0.0.11";
```
add `ocaml-ng` to arguments

---

Thank you for pointing out. I have fixed it.